set.mm

```diff
@@ -529946,6 +530087,137 @@ every nonzero prime ideal contains a prime element (this is a
       MNOPQRWHWIWJVHVIVJ $.
   $}
 
+  ${
+    fldextrspun.k $e |- K = ( L |`s F ) $.
+    fldextrspun.i $e |- I = ( L |`s G ) $.
+    fldextrspun.j $e |- J = ( L |`s H ) $.
+    fldextrspun.2 $e |- ( ph -> L e. Field ) $.
+    fldextrspun.3 $e |- ( ph -> F e. ( SubDRing ` I ) ) $.
+    fldextrspun.4 $e |- ( ph -> F e. ( SubDRing ` J ) ) $.
+    fldextrspun.5 $e |- ( ph -> G e. ( SubDRing ` L ) ) $.
+    fldextrspun.6 $e |- ( ph -> H e. ( SubDRing ` L ) ) $.
+    ${
+      fldextrspundglemul.7 $e |- ( ph -> ( J [:] K ) e. NN0 ) $.
+      fldextrspundglemul.1 $e |- E = ( L |`s ( L fldGen ( G u. H ) ) ) $.
+      $( Given two finite field extensions ` I / K ` and ` J / K ` of the same
+         field ` K ` , and the composiste field ` E = I J ` , the degree of the
+         extension of the composite field ` E / K ` is at ost the product of
```

Comment:
"at ost" -> "at most". Also maybe it's better to write "two finite extensions", since "finite field extensions" can confuse someone into thinking that the fields themselves are finite.